### PR TITLE
Update Building in Windows 10 or 11 with MSYS2.md

### PR DESCRIPTION
**Pull request by pwnept**
Changed default git command to checkout with a tag instead of a branch, kept the old command modified as a comment. (because i.e. release_5.0.0 branch does not exist)
Added where to find tags.
Updated version numbers of iNav and ARM toolchain.

### CODE REVIEW

File: docs/development/Building in Windows 10 or 11 with MSYS2.md [hunk 1]
```diff
@@ -54,21 +61,21 @@ This will give you the version you need for any given release or master branch.
 
 https://github.com/xpack-dev-tools/arm-none-eabi-gcc-xpack/releases/
 ```
-# for version 2.6.1, version needed is 9.2.1
-wget https://github.com/xpack-dev-tools/arm-none-eabi-gcc-xpack/releases/download/v9.2.1-1.1/xpack-arm-none-eabi-gcc-9.2.1-1.1-win32-x64.zip
-unzip xpack-arm-none-eabi-gcc-9.2.1-1.1-win32-x64.zip
+# for iNav version 5.0.0, tookchain version needed is 10.2.1
```
Comment (DzikuVx) [inline docs/development/Building in Windows 10 or 11 with MSYS2.md]: Let's use `INAV` everywhere. All capitals please :)

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: docs/development/Building in Windows 10 or 11 with MSYS2.md (hunk 1 above)
Comment (erstec) [reply docs/development/Building in Windows 10 or 11 with MSYS2.md]: Exactly!
And please correct "tookchain"

Comment (pwnept) [reply docs/development/Building in Windows 10 or 11 with MSYS2.md]: Done!

**Review by pwnept — COMMENTED** [review]
Fixed in [b48adf5](https://github.com/iNavFlight/inav/pull/8248/commits/b48adf56b90d24e302b85b61791648c241771d9a)

**Review by DzikuVx — APPROVED** [review]

**Comment by MrD-RC** [comment]
INAV should be iNav. Look at the name of the repository iNavFlight.

**Comment by DzikuVx** [comment]
@MrD-RC some time we decided it's `INAV` not to have any connotation with Apple. Organization is iNavFlight but INAV is INAV

**Comment by MrD-RC** [comment]
@DzikuVx lower case i is not just Apple. Plenty of other companies also use the format.

I'm not a fan of the uppercase, shouting version. The original was iNavFlight inav.

**Pull request merged**